src/sordid-dragon.jquery.coffee

```diff
@@ -61,20 +61,32 @@ do ($=jQuery) ->
     # create a ghost under the cursor for us. That doesn't happen with touch
     # events, so we create one ourselves.
     $ghost = null
+    $ghostContainer = $("<div />")
+      .addClass("ghost-container")
+      .css
+        display: "none"
+        position: "fixed"
```

Comment:
@adam-h I actually think we can do away with the `$ghostContainer` and just let the `$ghost` element free float at the `body` root.